third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -791,8 +791,9 @@ class BlockedToWMMA : public OpRewritePattern<tt::DotOp> {
     auto warpsPerTile = warpsPerTileWMMA(dotOp, retShape, numWarps);
 
     auto CTALayout = ttg::getCTALayout(oldRetEncoding);
-    wmmaEnc = ttg::AMDWmmaEncodingAttr::get(ctx, wmmaVersion, warpsPerTile,
-                                            CTALayout);
+    bool isTransposed = (wmmaVersion == 2);
```

Comment:
Please add comments to explain why setting it like this way. I assume it's similar to the transpose bits in `BlockToMFMA`.

Reply:
I will leave this to @giuseros :)
I borrowed this heuristic from his prototype

Reply:
Well, I cared mostly in adding support for wmma2 transposed layout. I am not 100% sure that, for the wmma1 layout, is possible to have a transposed version, but I didn't put much thought into it

Reply:
I believe that transposed version 1 is easy to implement, but main question for me is "do we need it"?
As far as I remember, it is hard or impossible to bypass LDS between dots with v1

Reply:
Yeah, this is my point. Can we not go through lds if we do tranposed layout? From what I remember and from what you are saying, it does not seem possible. In that case, I agree is not useful. Maybe you could put a comment about this?

Reply:
good idea, added an IR verifier: https://github.com/triton-lang/triton/pull/5456/files#diff-c05cf3aed297bf0c5f1296cc40c522b00fb300c7a4340a1f6be5b0bbe2c42039R1894